### PR TITLE
Fight simulator V1.2

### PR DESCRIPTION
**Pull request by P4uton**

### CODE REVIEW

File: src/views/fightSimulator.vue [hunk 1]
```diff
@@ -2,55 +2,10 @@
...
-            </thead>
-        </table>
-        <br>
-        <br>
-        <br>
         <div>
+            <ol>
+
+
             Note:
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]:
```suggestion
            Notes:
```

File: src/views/fightSimulator.vue [hunk 2]
```diff
@@ -2,55 +2,10 @@
...
-            </tr>
-            </thead>
-        </table>
-        <br>
-        <br>
-        <br>
         <div>
+            <ol>
+
+
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]: remove one empty line

File: src/views/fightSimulator.vue [hunk 3]
```diff
@@ -946,6 +960,29 @@ export default {
...
+        },
+
+        //format numbers
+        format(x){
+            return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
+        },
+
+
+
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]: remove two empty lines

File: src/views/fightSimulator.vue [hunk 4]
```diff
@@ -969,6 +1008,8 @@ export default {
             });
         },
 
+
+
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]: remove one empty line

File: src/views/fightSimulator.vue [hunk 5]
```diff
@@ -946,6 +960,29 @@ export default {
 
         },
 
+        displayFightOutcome(){
+            let text = document.getElementById('Difference');
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]:
```suggestion
            const text = document.getElementById('Difference');
```

File: src/views/fightSimulator.vue [hunk 6]
```diff
@@ -946,6 +960,29 @@ export default {
...
+            let value = parseFloat(text.innerHTML);
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]:
```suggestion
            const value = parseFloat(text.innerHTML);
```

File: src/views/fightSimulator.vue [hunk 7]
```diff
@@ -946,6 +960,29 @@ export default {
...
+        },
+
+        //format numbers
+        format(x){
+            return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
+        },
+
+
+
+        //not used anymore
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue:1000]: can be removed if not used, or is there a reason to keep them (maybe for later use or so)?

same for the next two methods

File: src/views/fightSimulator.vue [hunk 8]
```diff
@@ -901,17 +902,30 @@ export default {
...
                 let text = document.getElementById('defDeath' + i.toString());
-                text.innerHTML = defensiveDeathArray[i];
+                text.innerHTML = this.format(defensiveDeathArray[i]);
             }
 
         },
 
+        fillDefensiveDeathsWithHealing(defensiveDeathArray){
+            for (let i = 0; i < defensiveDeathArray.length; i++) {
+                let healing = document.getElementById("healing").value/100;
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]:
```suggestion
                const healing = document.getElementById("healing").value/100;
```

File: src/views/fightSimulator.vue [hunk 9]
```diff
@@ -901,17 +902,30 @@ export default {
...
+
+                let text = document.getElementById('defDeath' + i.toString());
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]:
```suggestion
                const text = document.getElementById('defDeath' + i.toString());
```

File: src/views/fightSimulator.vue [hunk 10]
```diff
@@ -634,9 +633,12 @@ export default {
             this.fillOffensiveDeaths(offensiveDeathArray);
 
             this.fillTable(totalAttack, totalDefense);
-            let strengthLost = this.calc_StrengthLost(offensiveDeathArray);
-            this.fillStrengthLost(strengthLost);
-        },
+            this.displayFightOutcome();
+            //let strengthLost = this.calc_StrengthLost(offensiveDeathArray);
+
+            //this.fillStrengthLost(strengthLost);
```
Comment (ArminWindlin) [inline src/views/fightSimulator.vue]: can be removed?

**Review by ArminWindlin — CHANGES_REQUESTED** [review]
Cool improvements!

Maybe change the name of the README.md you created, since it's not actually a readme but more development notes for you?

File: src/views/fightSimulator.vue (hunk 7 above)
Comment (P4uton) [reply src/views/fightSimulator.vue:1000]: I want to keep them for potential future use

**Review by ArminWindlin — APPROVED** [review]

**Pull request merged**